MHRA/products
# API Pod - Rust server for API

# API Pod - Rust server for API

[Airtable ticket](https://airtable.com/tbl7CISHKkr8Kdeh2/viwlDIU6cp8cbQYhj/reciuF4EBcHOrV675?blocks=hide)

![](https://media.giphy.com/media/B8l4KbEPvCpUI/giphy.gif)

### Acceptance Criteria

- [x] Rust server created as a base for API server
- [x] `healthz` returns **200**
### Testing information

1. Git pull this branch
2. Go to `/medicines/api`
3. Run `cargo run`
4. Once is compile open a browser tab and go to `http://localhost:8080/healthz`
5. You should see **OK** rendered on the page


### Pre-flight Checklist

~DUXD review approval~
~Accessibility Reviewed~

### Pre-merge Checklist
